apps/web/store/archive-queue.ts

```
@@ -1,103 +1,146 @@
+export const queueAtom = atomWithStorage(
+  "gmailActionQueue",
+  { activeThreads: {}, totalThreads: 0 },
+  createStorage(),
+  { getOnInit: true },
+);
 
-type ActionFunction = (threadId: string, ...args: any[]) => Promise<any>;
+type ActionFunction = (
+  threadId: string,
+  labelId?: string,
+) => Promise<ServerActionResponse<{}>>;
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)